### DISCUSSION TITLE
Add new configuration files for MaskRCNN algorithm in mmdetection. 给 mmdet MaskRCNN 算法增加剩余的新配置文件

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#491] [MMDetection]
# Add new configuration files for MaskRCNN algorithm in mmdetection. 
 给 mmdet MaskRCNN 算法增加剩余的新配置文件 

According to the [Config adaptation document]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc))  and 
 the supported algorithm configurations, add all configurations 
 for MaskRCNN and verify the correctness of inference and training.

根据 [Config适配文档]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc))  内容和已经支持的算法配置，给 MaskRCNN 所有配置
 新增配置并验证推理和训练正确性

## Task Information

**Repository**：MMDetection

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：ZhaoCake

**Claim Time**：2023年08月28日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/10905

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by ZhaoCake** [comment]
https://github.com/open-mmlab/mmdetection/pull/10905
